### PR TITLE
add(docs): clarification on setup locally md

### PR DESCRIPTION
**Pull request by coreytan00**
Clarification to running commands through the root directory `freeCodeCamp.`

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: docs/how-to-setup-freecodecamp-locally.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ If you are on a different OS and/or are still running into issues, reach out to
 ### Configuring dependencies
 
 #### Step 1: Setup the environment variable file
-The default API keys and environment variables are stored in the file `sample.env`.  This file needs to be copied to a new file named `.env` that is accessed dynamically during the installation step.
+Within the root directory `freeCodeCamp`, the default API keys and environment variables are stored in the file `sample.env`.  This file needs to be copied to a new file named `.env` that is accessed dynamically during the installation step. From here moving forward, all commands should be run from the root directory.
```
Comment (xyozio) [inline docs/how-to-setup-freecodecamp-locally.md]: There is a double space before `This file needs to...`

File: docs/how-to-setup-freecodecamp-locally.md [hunk 2]
```diff
@@ -160,7 +160,7 @@ If you are on a different OS and/or are still running into issues, reach out to
 ### Configuring dependencies
 
 #### Step 1: Setup the environment variable file
-The default API keys and environment variables are stored in the file `sample.env`.  This file needs to be copied to a new file named `.env` that is accessed dynamically during the installation step.
+Within the root directory `freeCodeCamp`, the default API keys and environment variables are stored in the file `sample.env`. This file needs to be copied to a new file named `.env` that is accessed dynamically during the installation step. From here moving forward, all commands should be run from the root directory.
```
Comment (ojeytonwilliams) [inline docs/how-to-setup-freecodecamp-locally.md]:
```suggestion
The default API keys and environment variables are stored in the file `sample.env`, found in the root directory, `freeCodeCamp`. This file needs to be copied to a new file named `.env` that is accessed dynamically during the installation step. From here moving forward, all commands should be run from the root directory.
```
The previous wording sounded a little odd - as if 'within the root directory' the keys are stored in `sample.env`, but outside the root directory they're stored somewhere else.  For example "Within London beer is stored in casks" implies that outside of London it isn't.

Possibly I'm reading too much into this!

**Review by ojeytonwilliams — COMMENTED** [review]
I'm not sure if I'm being overly picky, here. If someone wants to merge as is, go right ahead.

**Comment by raisedadead** [comment]
Hi @coreytan00 thanks for the PR. We need to draw a line between adding too much information and just enough information. IMHO, sometimes too much information is always confusing.

It rather alright to ask questions and learn while working on the project. This particular addition seems like too much information to me.

**Pull request closed without merging**